research/2018-6-wb5-exploration-1.md

```diff
@@ -194,6 +194,14 @@ Using the same modules as the carousel in V4, along with a single new module, th
 * [Working Example](2018-assets/6-component-v5/docs/countdown-test.html)
 * [Source code](https://github.com/wet-boew/wet-boew-documentation/tree/master/research/2018-assets/6-component-v5/)
 
+### Component Prototype 6
```

Comment:
Can you move that block in this summary page instead

https://wet-boew.github.io/wet-boew-documentation/research/2018-14-wet5-summary.html

Thanks